<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>654</width>
    <height>382</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="lblShowImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>65</y>
      <width>255</width>
      <height>255</height>
     </rect>
    </property>
    <property name="lineWidth">
     <number>1</number>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="btnReset">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>291</y>
      <width>271</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Reset</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldZ">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>245</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_7">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>171</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Point (x)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>97</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Pitch (deg)</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldYaw">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>134</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblPitch">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>97</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblX">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>171</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>134</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Yaw (deg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="lblY">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>208</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sldPitch">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>97</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QSlider" name="sldRoll">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>60</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblZ">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>245</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="lblYaw">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>134</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSlider" name="sldY">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>208</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="lblRoll">
    <property name="geometry">
     <rect>
      <x>612</x>
      <y>60</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_8">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>208</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Point (y)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_9">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>245</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Point (z)</string>
    </property>
   </widget>
   <widget class="QSlider" name="sldX">
    <property name="geometry">
     <rect>
      <x>402</x>
      <y>171</y>
      <width>202</width>
      <height>25</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>60</y>
      <width>67</width>
      <height>25</height>
     </rect>
    </property>
    <property name="text">
     <string>Roll (deg)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>111</width>
      <height>131</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap resource="rsc.qrc">:/icons/CoordinateSystem.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>654</width>
     <height>30</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionLoad"/>
    <addaction name="actionSave"/>
    <addaction name="actionSave_as"/>
    <addaction name="actionPrint"/>
    <addaction name="actionClose"/>
   </widget>
   <widget class="QMenu" name="menuGenerate_Phantom">
    <property name="title">
     <string>Generate Phantom</string>
    </property>
    <addaction name="actionR"/>
    <addaction name="actionX"/>
    <addaction name="actionY"/>
    <addaction name="actionZ"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionInstruction"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuGenerate_Phantom"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionGradiant_along_X_axis">
   <property name="text">
    <string>Gradiant along X axis</string>
   </property>
  </action>
  <action name="actionGradiant_along_Y_axis">
   <property name="text">
    <string>Gradiant along Y axis</string>
   </property>
  </action>
  <action name="actionGradiant_along_Z_axis">
   <property name="text">
    <string>Gradiant along Z axis</string>
   </property>
  </action>
  <action name="actionInformatio">
   <property name="text">
    <string>Informatio</string>
   </property>
  </action>
  <action name="actionR">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Radial Gradiant</string>
   </property>
  </action>
  <action name="actionX">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradiant along x axis</string>
   </property>
  </action>
  <action name="actionY">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradiant along y axis</string>
   </property>
  </action>
  <action name="actionZ">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>Gradient along z axis</string>
   </property>
  </action>
  <action name="actionInstruction">
   <property name="text">
    <string>Instruction</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="text">
    <string>Save as</string>
   </property>
  </action>
  <action name="actionPrint">
   <property name="text">
    <string>Print</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="rsc.qrc"/>
 </resources>
 <connections/>
</ui>
